site2/website/blog/2021-08-xx-Apache-Pulsar-2-7-3.md

```diff
@@ -0,0 +1,139 @@
+
+- **Resolution**: Implemented `topicName.getDomain().value()` rather than hard coding `persistent.` Now you can use the discovery service for a partitioned, non-persistent topic successfully.
+
+### Other connectors can now use the Kinesis `Backoff` class. [PR-10744](https://github.com/apache/pulsar/pull/10744)
+
+- **Issue**: The Kinesis sink connector `Backoff` class in the Pulsar client implementation project in combination  with the dependency `org.apache.pulsar:pulsar-client-original` increased the connector size. 
+
+- **Resolution**: Added a new class `Backoff` in the function io-core project so that the Kinesis sink connector and other connectors can use the class.
+
+## Client
+
+### A `FLOW` request with zero permits can not be sent. [PR-10506](https://github.com/apache/pulsar/pull/10506)
+
+- **Issue**: When a broker received a `FLOW` request with zero permits, an exception was thrown and then the connection was closed. This triggered frequent reconnections and caused duplicate or out-of-order messages. 
+  
+- **Resolution**: Added a validation that verifies the permits of a `FLOW` request before sending it. If the permit is zero, the `FLOW` request can not be sent.
+
+## Function and connector
+
+### The Kinesis sink connector acknowledges successful messages. [PR-10769](https://github.com/apache/pulsar/pull/10769)
+
+- **Issue**: The Kinesis sink connector did not acknowledge messages after they were sent successfully. 
+  
+- **Resolution**: Added acknowledgement for the Kinesis sink connector once a message is sent successfully. 
+ 
+## Docker
+
+### Function name length cannot exceed 52 characters when using Kubernetes runtime. [PR-10531](https://github.com/apache/pulsar/pull/10531)
+
+- **Issue**: When using Kubernetes runtime, if a function was submitted with a valid length (less than 55 characters), a StatefulSet was created but it was unable to spawn pods. 
+  
+- **Resolution**: Changed the maximum length of a function name from 55 to 53 characters for Kubernetes runtime. With this fix, the length of a function name can not exceed 52 characters. 
+
+## Dependency 
+
+### `pulsar-admin` connection to proxy is stable when TLS is enabled. [PR-10907](https://github.com/apache/pulsar/pull/10907)
+
+- **Issue**: `pulsar-admin` was unstable over the TLS connection because of the Jetty bug in SSL buffering introduced in Jetty 9.4.39. It caused large function jar uploads to fail frequently.
+  
+- **Resolution**: Upgraded Jetty to 9.4.42.v20210604, so that `pulsar-admin` connection to proxy is stable when TLS is enabled.
+
+# What is Next?
+
+If you are interested in learning more about Pulsar 2.7.3, you can [download 2.7.3](https://pulsar.apache.org/en/versions/) and try it out now! 
+
+Do you have a Pulsar story to share? Join us and speak at Pulsar Virtual Summit
+Europe 2021! Feel free to [submit an abstract today](https://streamnative.io/en/blog/community/2021-06-30-announcing-pulsar-virtual-summit-europe-2021-cfp-is-open)!
```

Comment:
the link "https://streamnative.io/en/blog/community/2021-06-30-announcing-pulsar-virtual-summit-europe-2021-cfp-is-open" does not work, probably there is some typo.

can we add a link to the CFP on the official Pulsar Summit  website https://pulsar-summit.org/ ?

Reply:
- Try this link: https://streamnative.io/en/blog/community/2021-06-30-announcing-pulsar-summit-europe-2021-cfp-is-open/
- It's better to use the official Pulsar Summit website. Maybe we can add the above announcement to Pulsar blog as well and use the link.

Reply:
Thanks @eolivelli @Jennifer88huang 
I think https://streamnative.io/en/blog/community/2021-06-30-announcing-pulsar-summit-europe-2021-cfp-is-open/ is better since it contains:
- CFP link
- more info than the info on the CFP page